File: C++/1005.cpp
#include <iostream>
#include <iomanip>

using namespace std;

#define WEIGHT_A 3.5
#define WEIGHT_B 7.5

int main() {
    double scoredA, scoredB, avgAB;

    cin >> scoredA;
    cin >> scoredB;

    avgAB = ((scoredA * WEIGHT_A) + (scoredB * WEIGHT_B)) / (WEIGHT_A + WEIGHT_B);

    cout << "MEDIA = " << fixed << setprecision(5) << avgAB << '\n';

    return 0;
}

File: Go/1005.go
package main

import "fmt"

const (
	weightA = 3.5
	weightB = 7.5
)

func main() {
	var A, B float64

	fmt.Scan(&A)
	fmt.Scan(&B)

	fmt.Printf("MEDIA = %.5f\n", (A*weightA+B*weightB)/(weightA+weightB))
}
